test/brain/redis_store_test.exs

```diff
@@ -0,0 +1,105 @@
+defmodule Elix.RedisStore.RedisStoreTest do
+  use ExUnit.Case, async: true
+  alias Elix.Brain.RedisStore
+
+  setup do
+    RedisStore.start_link
+    Redix.command!(:redis, ["FLUSHDB"])
```

Comment:
Maybe this should be a function on `RedisStore`? Feels a little awkward reaching down into its Redix dependency directly.